packages/config/src/api/site_info.js

```diff
@@ -71,3 +79,25 @@ const getAddons = async function (api, siteId) {
     throwUserError(`Failed retrieving addons for site ${siteId}: ${error.message}. ${ERROR_CALL_TO_ACTION}`)
   }
 }
+
+const getIntegrations = async function ({ api, ownerType, ownerId, logs }) {
+  if (ownerId === undefined) {
+    return []
+  }
+
+  try {
+    const token = api.accessToken()
+    const response = await fetch(`https://api.netlifysdk.com/${ownerType}/${ownerId}/integrations`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+
+    const integrations = await response.json()
+    return Array.isArray(integrations) ? integrations : []
```

Comment:
I'm curious if there are any plans to make the response always return a specific format regardless of the number of entries it contains, so that consumers don't have to do this type of thing?

Reply:
This was more around easily catching 404s etc, and just ensuring a valid response is always piped back.

I confess - copilot cooked this up, and then I liked it so stuck with it

Reply:
the response is always an array however 🎉

Reply:
Ah, I read this wrong! I thought you were doing `Array.isArray(integrations) ? integrations : [integrations]` because the response may or may not be an array, but I now realise that's not the case.

Hmm, I'd say that doing this to assert the success of the request feels a bit odd. You could use the status code?

Reply:
Yeah potentially, it's a bit more explicit too

Reply:
Ah, I see why copilot cooked that up, all the methods in the file take the same approach, will keep it in for consistency